tools/xsetwacom.c

```diff
@@ -1442,17 +1447,20 @@ static Bool parse_actions(Display *dpy, int argc, char **argv, unsigned long* da
  * @param  argc        Number of command line arguments we've been passed
  * @param  argv        Command line arguments we need to parse
  */
-static void special_map_property(Display *dpy, XDevice *dev, Atom btnact_prop, unsigned int offset, int argc, char **argv)
+static int special_map_property(Display *dpy, XDevice *dev, Atom btnact_prop, unsigned int offset, int argc, char **argv)
 {
 	unsigned long *data, *btnact_data;
 	Atom type, prop = 0;
 	int format;
 	unsigned long btnact_nitems, bytes_after;
 	unsigned long nitems = 0;
+	int status = EXIT_SUCCESS;
```

Comment:
just a general comment, not necessary to apply in this patch: it's *usually* neater to default to the most common error and then use this pattern:
```c
  int status = EXIT_FAILURE;
  if (blah)
    goto out;
  if (!foo)
    goto out;
  if (bar) {
    status = EXIT_INVALID_USAGE;
  }
  status = EXIT_SUCCESS;
out:
  return status;
```
Reason being that you can use `goto out` as single line in most cases and you don't accidentally return success until you get to the end.